uint32 ps_id
uint16 ps_status
string ps_gender
string ps_age